kodi_addon_checker/check_dependencies.py

```diff
@@ -56,6 +56,9 @@ def check_addon_dependencies(report: Report, repo_addons: dict, parsed_xml, bran
                                    "version mismatch for optional dependencies: %s. Available: %s, Required: %s" %
                                    (dependency, deps_version, version))
 
+    if 'xbmc.python' not in deps:
```

Comment:
Not every add-on has to list `xbmc.python` as dependency, only python add-ons.

Reply:
Python is the only possible scripting language, but skins, languages, controllers, webinterfaces, ... don't use python. So they shouldn't (and also do not) import `xbmc.python`.